modules/heat_conduction/src/actions/ThermalContactAction.C

```diff
@@ -96,6 +96,13 @@ ThermalContactAction::validParams()
                                 "Offset to gap distance from secondary side");
   params.addParam<VariableName>("mapped_primary_gap_offset",
                                 "Offset to gap distance mapped from primary side");
+  params.addParam<bool>(
+      "check_boundary_restricted",
+      true,
+      "Whether to pass the option with this name to the AuxKernels that are used "
+      "for output to allow simulations to run in cases where multiple faces of an "
+      "element are in a contact surface. Output will only account for the contribution "
+      "from one of the faces in that case.");
```

Comment:
I'll look at what is in the AuxKernel description -- maybe it should be closer to that one. I think there is a little more that should be conveyed for the contact case, though.

Reply:
Regarding `quadrature=false`, the thinking there was that the integration procedure is set up to only account for a single face, so the solution would be wrong. We'd need to dig a little deeper to see how important that is. I think it's OK limiting this to `quadrature=true` since that's by far the most widely used scenario anyway.